diff -Nru zabbix-2.1.9.orig/frontends/php/include/classes/class.cconfigfile.php zabbix-2.1.9/frontends/php/include/classes/class.cconfigfile.php
--- zabbix-2.1.9.orig/frontends/php/include/classes/class.cconfigfile.php	2013-11-05 01:35:47.000000000 +0900
+++ zabbix-2.1.9/frontends/php/include/classes/class.cconfigfile.php	2013-11-05 02:51:00.000000000 +0900
@@ -24,7 +24,7 @@
 	const CONFIG_NOT_FOUND = 1;
 	const CONFIG_ERROR = 2;
 
-	const CONFIG_FILE_PATH = '/conf/zabbix.conf.php';
+	const CONFIG_FILE_PATH = '/etc/zabbix/web/zabbix.conf.php';
 
 	public $configFile = null;
 	public $config = array();
diff -Nru zabbix-2.1.9.orig/frontends/php/include/classes/core/ZBase.php zabbix-2.1.9/frontends/php/include/classes/core/ZBase.php
--- zabbix-2.1.9.orig/frontends/php/include/classes/core/ZBase.php	2013-11-05 01:35:47.000000000 +0900
+++ zabbix-2.1.9/frontends/php/include/classes/core/ZBase.php	2013-11-05 02:53:27.000000000 +0900
@@ -272,7 +272,7 @@
 	 * @throws Exception
 	 */
 	protected function setMaintenanceMode() {
-		require_once $this->getRootDir().'/conf/maintenance.inc.php';
+		require_once '/etc/zabbix/web/maintenance.inc.php';
 
 		if (defined('ZBX_DENY_GUI_ACCESS')) {
 			$user_ip = (isset($_SERVER['HTTP_X_FORWARDED_FOR']) && !empty($_SERVER['HTTP_X_FORWARDED_FOR']))
@@ -288,7 +288,7 @@
 	 * Load zabbix config file.
 	 */
 	protected function loadConfigFile() {
-		$configFile = $this->getRootDir().CConfigFile::CONFIG_FILE_PATH;
+		$configFile = CConfigFile::CONFIG_FILE_PATH;
 		$config = new CConfigFile($configFile);
 		$this->config = $config->load();
 	}
diff -Nru zabbix-2.1.9.orig/frontends/php/include/setup.inc.php zabbix-2.1.9/frontends/php/include/setup.inc.php
--- zabbix-2.1.9.orig/frontends/php/include/setup.inc.php	2013-11-05 01:35:47.000000000 +0900
+++ zabbix-2.1.9/frontends/php/include/setup.inc.php	2013-11-05 02:55:55.000000000 +0900
@@ -429,7 +429,7 @@
 	function stage6() {
 		$this->setConfig('ZBX_CONFIG_FILE_CORRECT', true);
 
-		$config = new CConfigFile(Z::getInstance()->getRootDir().CConfigFile::CONFIG_FILE_PATH);
+		$config = new CConfigFile(CConfigFile::CONFIG_FILE_PATH);
 		$config->config = array(
 			'DB' => array(
 				'TYPE' => $this->getConfig('DB_TYPE'),
@@ -496,7 +496,7 @@
 		$this->HIDE_CANCEL_BUTTON = !$this->DISABLE_NEXT_BUTTON;
 
 
-		$table = array('Configuration file', BR(), '"'.Z::getInstance()->getRootDir().CConfigFile::CONFIG_FILE_PATH.'"',
+		$table = array('Configuration file', BR(), '"'.CConfigFile::CONFIG_FILE_PATH.'"',
 			BR(), 'created: ', $this->getConfig('ZBX_CONFIG_FILE_CORRECT', false)
 			? new CSpan(_('OK'), 'ok')
 			: new CSpan(_('Fail'), 'fail')
@@ -509,7 +509,7 @@
 				? array($error_text, BR(), 'Please install it manually, or fix permissions on the conf directory.', BR(), BR(),
 					'Press the "Download configuration file" button, download the configuration file ',
 					'and save it as ', BR(),
-					'"'.Z::getInstance()->getRootDir().CConfigFile::CONFIG_FILE_PATH.'"', BR(), BR(),
+					'"'.CConfigFile::CONFIG_FILE_PATH.'"', BR(), BR(),
 					new CSubmit('save_config', 'Download configuration file'),
 					BR(), BR()
 				)
